xcode/Subconscious/Shared/Library/SQLite3Database.swift

```diff
@@ -1,16 +1,6 @@
 import SQLite3
 import Foundation
 
-extension Array {
```

Comment:
Remove this extension. It assumes too much, and we were relying on it in places we shouldn't have (like tests), causing complection in places we shouldn't be.